### PR TITLE
Added readme

### LINKED ISSUE #34: make readme
maybe provide tips for people to edit this thing :)

### PR DESCRIPTION
**Pull request by GeorgeWangnottaken**
Hi Savo, 

Here's a smol website readme. I'd appreciate any feedback on the content!

### CODE REVIEW

**Review by savob — CHANGES_REQUESTED** [review]
Can you please add a link to the plugin you use, and explain the directory structure as we chatted about.

File: team_members.html [hunk 1]
```diff
@@ -58,6 +58,7 @@ <h3><b>Team Member List*</b><br></h3>
                 </div>
                 <br>
                 <ul class="column-setting long-copy"><!--Names here https://docs.google.com/spreadsheets/d/13bADKZJmOhvXR2PBIsjA773Bjn1bCj1U0bQnRbw5FrU/edit?usp=sharing-->
+                    <!-- <li>KAAIIIIIIIIIII (the cool one)</li> -->
```
Comment (savob) [inline team_members.html:61]: Hrmmmmmmmmm :thinking:

File: README.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ This is the code for the team website: https://hpvdt.ca/. There are a few groups
...
     * All photo files are available in resources/img except for the big header background images, which are in resources/css/cssimg
- 
   
 [comment]: <> (Will add how to update this website/test it)
 
-Contact me on the HPVDT Discord or at <georgez.wang@mail.utoronto.ca> if you have questions or stuff to add to the website!
\ No newline at end of file
+To update the code, we use Visual Studio Code. We preview the changes before making pull requests on GitHub with the Live Server extension. 
+
+Contact me on the HPVDT Discord or at <georgez.wang@mail.utoronto.ca> if you have questions or ideas to add to the website!
```
Comment (savob) [inline README.md]: Let's remove your personal email from this. We have a @hpvdt/webmasters tag that can be used to reach us or list one of the team emails instead.

**Review by savob — CHANGES_REQUESTED** [review]
There's something up with the main page regarding the exec photos. When the window is narrow it's okay, but when viewing full screen (at 1080p) they get seemingly randomly resized in front of or behind text.

File: README.md (hunk 2 above)
Comment (GeorgeWangnottaken) [reply README.md]: thanks for the feedback! the readme draft is legit enough for final review

**Review by savob — APPROVED** [review]
Looks good! 

Going to merge and delete this branch (so we can go back to having just a `dev` instead of `dev2`)

**Pull request merged**